### PR TITLE
vault-name-status-bar

### PR DESCRIPTION
**Pull request by 1C0D**
A very simple plugin for convenience

# I am submitting a new Community Plugin

## Repo URL

Link to my plugin:

## Release Checklist
- [ x] I have tested the plugin on
  - [x ]  Windows
  - [x ]  macOS
  - [x ]  Linux
  - [ x]  Android _(if applicable)_
  - [x ]  iOS _(if applicable)_
- [x ] My GitHub release contains all required files
  - [x ] `main.js`
  - [x ] `manifest.json`
  - [x ] `styles.css` _(optional)_
- [x ] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [ x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [ x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [ x] I have read the tips in https://github.com/obsidianmd/obsidian-releases/blob/master/plugin-review.md and have self-reviewed my plugin to avoid these common pitfalls.
- [x ] I have added a license in the LICENSE file.
- [ x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

### CODE REVIEW

**Comment by joethei** [comment]
Would recommend adding a screenshot to your readme.

[MyPlugin](https://github.com/1C0D/Obsidian-Vault-Name-in-Status-Bar/blob/759a4ac5ded9f2628d4ba90a2047753a57ae165c/src/main.ts#L3)
Rename this to something more meaningful.

[this.statusBarVaultName.remove;](https://github.com/1C0D/Obsidian-Vault-Name-in-Status-Bar/blob/759a4ac5ded9f2628d4ba90a2047753a57ae165c/src/main.ts#L14)
This is not needed, Obsdian handles it automatically.

**Comment by 1C0D** [comment]
ok done. thks for the suggestions

**Comment by 1C0D** [comment]
I added a little timer (50 ms) to be sure it's always position and a separator e.g **| Vault name**

**Comment by 1C0D** [comment]
@joethei Hi. Sorry I missed the point about changing the class name.
now "statusBarVaultName"

**Comment by 1C0D** [comment]
Is there a change I've forgotten to do? or something wrong ?

**Comment by 1C0D** [comment]
@liamcain there is this last one

**Comment by liamcain** [comment]
- [document.querySelector](https://github.com/1C0D/Obsidian-Vault-Name-in-Status-Bar/blob/658ee5e2b61fd6b661b52fef567487e1d04e3dd7/src/main.ts#L41) no need to query the DOM, you can just do `this.title.detach()`.
- [style.fontSize](https://github.com/1C0D/Obsidian-Vault-Name-in-Status-Bar/blob/658ee5e2b61fd6b661b52fef567487e1d04e3dd7/src/main.ts#L24) You should avoid assigning styles via javascript. Instead, just add a class to the element and move all these styles into CSS so that they are more easily adaptable by themes.
- [div .status-bar](https://github.com/1C0D/Obsidian-Vault-Name-in-Status-Bar/blob/658ee5e2b61fd6b661b52fef567487e1d04e3dd7/src/main.ts#L20) the "div" here isn't doing anything. This should just be .status-bar
- [innerHTML](https://github.com/1C0D/Obsidian-Vault-Name-in-Status-Bar/blob/658ee5e2b61fd6b661b52fef567487e1d04e3dd7/src/main.ts#L22) in general, you shoudn't really be using innerHTML here. It's a weird example, but if the user had a vault called "<script>" then it will actually create an html script tag meaning this is a vulnerability. You can just use innerText instead.

**Comment by 1C0D** [comment]
@Liamcain Can you review this, I did a better version with the name always before and expandable on click. And on option to have it reduced at start or not.
NameVaultStatusBar

**Comment by 1C0D** [comment]
I was surprised by your comment. I guess you can see changes I do on the repo. 
So I did the changes you suggested. And added the css to the release. much better to do some changes

**Comment by liamcain** [comment]
Did you remember to push your changes? I don't see anything different.

**Comment by 1C0D** [comment]
@liamcain I don't know what I did wrong. I deleted release 1.0.1 and did it again. I checked it on another vault and checked the js. It should be ok

**Comment by liamcain** [comment]
Looks good, can you rebase this pull request and move your entry to the bottom of the plugin list?

**Comment by liamcain** [comment]
@1C0D, just wanted to make sure you saw the earlier message.

**Comment by 1C0D** [comment]
this branch is not needed anymore. when I tried to the rebase, something went wrong and the plugin was not appearing in the releases anymore. So you told me to do a new PR and it was validated. I close this
